mantidproject/mantid
# Fix exception in fit browser

**Description of work.**
An exception has been reported multiple time in the error reporter relating to the plot guess feature in the fit property browser. The bug reports a ValueError when line.remove() is called, indicating that the line no longer exists. I think this could be due to removing the plot guess line in the figure options. Catching ValueErrors while calling .remove fixes this issue. 


**To test:**
Try before and after these changes

- Load a workspace
- Open the fit browser
- Add a function
- Right click and press plot
- Open figure options and remove the line corresponding to the function
- Now try change a parameter function or right click and remove the plot. There should now be no exception thrown.




*There is no associated issue.*






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
